platform/linux_x64/docker/Dockerfile

```
@@ -0,0 +1,22 @@
+FROM debian:latest
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN groupadd \
+  --gid 114 \
+  engflow && \
+  useradd \
+  --home-dir /home/engflow \
```

```suggestion
  --home-dir /var/lib/engflow \
```
?

And also below.

(I guess not, since this container is for actions, not the worker itself, but flagging it just in case.)

---

Let's actually keep this the same (but let me know if you feel strongly, I'll send a follow-up).

The path shouldn't actually matter either way. Nothing should use `HOME`, but occasionally actions want to write cache or config files into the home directory. They'll disappear when we shut down the container, so it's more or less equivalent to `/tmp` in terms of persistence.

`/home/engflow` seems like a slightly less weird location for a home directory than `/var/lib`, which makes more sense for a service. Either should work, but in case there are any oddball actions out there that assume `/home`, perhaps that's more likely to work.